Formula/gcc.rb

```diff
@@ -89,6 +90,7 @@ def install
       --with-pkgversion=#{pkgversion}
       --with-bugurl=#{tap.issues_url}
     ]
+    args << "--enable-libphobos" if Hardware::CPU.intel?
```

Comment:
Can we leave a comment here that this is part of `gdc`? Someone might try to add `gdc` support to ARM in the future but not realise that this will also be needed for that.

Reply:
CI won't finish for a while. The Intel runners are currently offline for maintenance.

Reply:
thank you for nice info :)